library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/CheckedTextViewMapper.kt

```diff
@@ -43,18 +44,29 @@ internal class CheckedTextViewWireframeMapper(
         return stringUtils.formatColorAndAlphaAsHexa(view.currentTextColor, OPAQUE_ALPHA_VALUE)
     }
 
-    override fun resolveCheckBoxSize(view: CheckedTextView, pixelsDensity: Float): Long {
+    override fun resolveCheckBoxBounds(view: CheckedTextView, pixelsDensity: Float): GlobalBounds {
+        val viewGlobalBounds = resolveViewGlobalBounds(view, pixelsDensity)
+        val textViewPaddingRight =
+            view.totalPaddingRight.toLong().densityNormalized(pixelsDensity)
```

Comment:
Seems not quite RTL compatible? Maybe we should leave the comment / TODO here?

Reply:
yes is not, we will leave this for later but we have it in mind. That will be another problem to solve at some point as none of our SR content is RTL compatible for now :)